bundle/jsky.app.ot/src/main/scala/jsky/app/ot/vcs/VcsOtClient.scala

```diff
@@ -20,6 +20,8 @@ object VcsOtClient {
   def ref_=(c: Option[VcsOtClient]): Unit = {
     client = c
   }
+
+  def unsafeGetRegistrar: VcsRegistrar = ref.get.reg
```

Comment:
I would prefer `ref.fold(sys.error("some useful message"))(_.reg)` since the `None.get` message isn't very useful.